life/original/22-chapter-22.adoc

```diff
@@ -1729,3 +1834,5 @@ and give an easier motion to the administration of affairs; however,
 as it is some men`'s duty to plough, some to sow, some to water, and some to reap,
 so it is the wisdom as well as duty of a man, to yield to the mind of Providence,
 and cheerfully, as well as carefully, embrace and follow the guidance of it.
+
+====
```

Comment:
same thing here... the embedded chunk needs to be terminated with a -- even though it is the end of a chapter. 👍🏻